packages/bot-web-ui/src/components/bot-notification/bot-notification.scss

```diff
@@ -1,15 +1,24 @@
-.bot-snackbar {
+.bot-notification {
     position: fixed;
     z-index: var(--zindex-snackbar);
-    right: 38rem;
-    top: 12rem;
+    left: 3.5rem;
```

Comment:
@farabi-deriv Bhai can you try `inset-inline-start` or `inset-inline-end` instead of `left`? Because soon we will be having RTL language support and it will break then.

Reply:
@shafin-deriv Sure. Updated it ✅